presto-hive/src/main/java/io/prestosql/plugin/hive/metastore/HiveMetastore.java

```diff
@@ -45,6 +45,8 @@
 
     List<String> getAllTables(String databaseName);
 
+    List<String> getTablesMatchingParameter(String databaseName, String parameterKey, String parameterValue);
```

Comment:
"matching" may suggest some pattern matching which is not the case here.
maybe call this `getTablesWithParameter`.